projects/extension/sql/idempotent/008-embedding.sql

```diff
@@ -47,6 +47,28 @@ $func$ language sql immutable security invoker
 set search_path to pg_catalog, pg_temp
 ;
 
+-------------------------------------------------------------------------------
+-- embedding_voyageai
+create or replace function ai.embedding_voyageai
+( model text
+, dimensions int
+, truncate boolean default true
+, api_key_name text default 'VOYAGE_API_KEY'
+) returns jsonb
+as $func$
+    select json_object
+    ( 'implementation': 'voyageai'
+    , 'config_type': 'embedding'
+    , 'model': model
+    , 'dimensions': dimensions
+    , 'truncate': truncate
```

Comment:
I decided that I want to add the `input_type` field to this config object too, so I will do that.

Reply:
Done now.